### ISSUE TITLE
Report logging does not work?..

### ISSUE THREAD

**Issue by spezialist1** [samilliken/openDCIM#650]
Hi all!

Actually, a question in a subject (see the picture).

![report_logging_err](https://cloud.githubusercontent.com/assets/10425179/8954787/a1e819f4-35f0-11e5-96c8-566f269e93cd.png)

I so understand, this action log even in development?

Very thanks for the support.

**Comment by samilliken** [comment]
There may be some other site-specific problem, as this works fine on my production install.

**Comment by wilpig** [comment]
That looks like a javascript error. Check the console

**Comment by spezialist1** [comment]
Wilbur, I checked everything twice: I downloaded the current openDCIM version from GitHub and I checked table fac_GenericLog correctness (the table size no more 500kB). Above the specified bug repeated. Besides, I have a colleague who also uses in work of openDCIM and has the same bug.

Very thanks for the support.

P.S.: I checked the console, I didn't find anything suspicious...

**Comment by MorinPierre** [comment]
I have the same issue.

**Comment by wilpig** [comment]
![screen shot 2015-08-03 at 8 33 38 am](https://cloud.githubusercontent.com/assets/1653882/9037554/6edad2a4-39ba-11e5-9cfe-989d0daba585.png)

I just double checked all the code for possible problems and not finding any in multiple browsers and setups.

**Comment by wilpig** [comment]
Are you running on windows?

**Comment by MorinPierre** [comment]
I'm on redhat.

**Comment by wilpig** [comment]
![screen shot 2015-08-03 at 9 00 40 am](https://cloud.githubusercontent.com/assets/1653882/9037937/271cf998-39be-11e5-88c7-279b15ede5fd.png)

Tested on a clean windows install.  I'll be closing this bug unless a verifiable problem can be reproduced.

**Comment by MorinPierre** [comment]
spezialist1, did you try with a clean database (without the PUE_DCEM modifications)?

**Comment by wilpig** [comment]
![screen shot 2015-08-03 at 9 13 24 am](https://cloud.githubusercontent.com/assets/1653882/9038135/fce934be-39bf-11e5-9b47-e8c4b8099a20.png)

Again no problems here.

**Comment by MorinPierre** [comment]
The problem comes from the modifications of the database in PUE_DCEM branch. If I use a standard database (without extra classes) everything works fine.
I think you can close this issue.

**Comment by spezialist1** [comment]
Hi all!

I double-checked on a clean DB from latest openDCIM-master and came to a conclusion that the bug appears only after these changes in the table fac_Config:

INSERT INTO fac_Config set Parameter="TimeInterval", Value="Last 7 Days", UnitOfMeasure="time", ValType="string", DefaultVal="Last 7 Days";

Any ideas?
Very thanks for the support.

**Comment by wilpig** [comment]
That is in no way related.  Not even close.

**Comment by spezialist1** [comment]
Hi all!

I again checked today everything and here that we have:
-- first, PUE_DCEM modifications doesn't influence presence at all or absence is higher the specified bug in report_logging.php;
-- secondly, the bug above specified appears only then when in a configuration of openDCIM to change value of the 'Time Zone' parameter to any other value other than 'America/Chicago'.

That is why both and at Morin Pierre this report doesn't work for me, and your Wilbur works. And that is why the bug appeared when I changed table fac_Config from clean openDCIM-master for own table fac_Config with other value of the 'timezone' parameter.

Wilbur, please, fix this bug of report_logging.php and the timezone in openDCIM.

Very thanks for the support.

**Comment by wilpig** [comment]
Opened a new issue for this as it potentially affects a few other reports.
